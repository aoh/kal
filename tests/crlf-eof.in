# flags: -t 0
every Thursday: foo w/ crlf
every Friday: bar sans eof newline