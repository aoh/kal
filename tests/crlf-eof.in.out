1.1.1970, Thursday, week 1
 - foo w/ crlf

2.1.1970, Friday, week 1
 - bar sans eof newline
